pkg/kubecost/asset.go

```diff
@@ -447,23 +447,23 @@ func (at AssetType) String() string {
 // Any is the most general Asset, which is usually created as a result of
 // adding two Assets of different types.
 type Any struct {
-	labels     AssetLabels
-	properties *AssetProperties
-	start      time.Time
-	end        time.Time
-	window     Window
-	adjustment float64
+	Labels     AssetLabels
+	Properties *AssetProperties
+	Start      time.Time
+	End        time.Time
+	Window     Window
+	Adjustment float64
```

Comment:
I'll be honest, this seems reasonable by design, but there are a _lot_ of cases which just ask for the list of Assets. In that way, I feel like it's actually pretty useful, and the interface actually _does_ fit. However, I think when you're only given a list of interfaces (without any sort of visitor pattern implemented), it can be really hard to work without some type of `byType` -- I think we can definitely improve on this as we go....

One thing to note is that for backwards compatibility reasons currently, we _can't_ just axe the Asset map :(